packages/cedar-amcharts/src/render/render.ts

```diff
@@ -67,8 +67,8 @@ export function fillInSpec(spec: any, definition: any) {
         spec.valueField = graph.valueField
 
         // Group vs. stack
-        if (!!series.group) {
-          graph.newStack = true
+        if (!!series.stack) {
```

Comment:
I think we should add ` && graph.newStack` to this `if` so we don't set this on other chart types (line, scatter, etc).